apps/core/client/queries/get-customer.ts

```diff
@@ -0,0 +1,61 @@
+import { cache } from 'react';
+
+import { client } from '..';
+import { graphql } from '../graphql';
+import { revalidate } from '../revalidate-target';
+
+const GET_CUSTOMER_QUERY = graphql(`
+  query getCustomer {
+    customer {
+      entityId
+      company
+      email
+      firstName
+      lastName
+      phone
+      formFields {
+        entityId
+        name
+        __typename
+        ... on CheckboxesFormFieldValue {
+          valueEntityIds
+          values
+        }
+        ... on DateFormFieldValue {
+          date {
+            utc
+          }
+        }
+        ... on MultipleChoiceFormFieldValue {
+          valueEntityId
+          value
+        }
+        ... on NumberFormFieldValue {
+          number
+        }
+        ... on PasswordFormFieldValue {
+          password
+        }
+        ... on TextFormFieldValue {
+          text
+        }
+      }
+    }
+  }
+`);
+
+export const getCustomer = cache(async (customerId: string) => {
+  const response = await client.fetch({
+    document: GET_CUSTOMER_QUERY,
+    fetchOptions: { next: { revalidate } },
```

Comment:
don't we need to use here `cache: 'no-store' `option  for Customer?

Reply:
good catch 👍 thanks